releases/release-1.31/release-notes/maps/pr-124075-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 124075
+releasenote:
+  text: 'DRA: client-side validation of a ResourceHandle would have accepted a missing
+    DriverName, whereas server-side validation then would have raised an error.'
```

Comment:
Is this explaining a bug fix?

Reply:
What does DRA stand for? Dynamic Resource Allocation?

Reply:
Yep, this is a bug fix. Maybe we can add "fix" and define DRA here.